docarray/index/backends/epsilla.py

```diff
@@ -60,10 +61,12 @@ def __init__(self, db_config=None, **kwargs):
                 db_path=self._db_config.db_path,
             )
 
-            if status_code != 200:
-                if status_code == 500 and (
-                    "already loaded" in response["message"]
-                    or "already exists" in response["message"]
+            if status_code != HTTPStatus.OK:
+                # Epsilla returns HTTP 500 when multiple clients connect to the same db
+                # Bug filed at https://github.com/epsilla-cloud/vectordb/issues/93
```

Comment:
if the quick is fix on Epsilla side, would it be possible to wait for a patch on epsille side?